subsys/fb/font_8x8.c

```diff
@@ -0,0 +1,2586 @@
+﻿/**********************************************/
```

Comment:
Missing license header

Reply:
this font is modified from linux kernel source, and there is not lincense in it see https://github.com/steev/linux-kernel/blob/master/drivers/video/console/font_8x8.c

Reply:
Is the linux kernel not GPL licensed maybe better look for a font with a less strict license?

Please have a look at https://docs.zephyrproject.org/latest/contribute/index.html#contributing-non-apache-2-0-licensed-components

Reply:
I update with the OFL 1.1 license font download from the https://www.fontsquirrel.com/fonts/download/abeezee